src/shared/varlink-idl.c

```diff
@@ -17,6 +17,9 @@ enum {
         _COLOR_MAX,
 };
 
+#define varlink_idl_log(error, format, ...) log_debug_errno(error, "Varlink-IDL: " format, ##__VA_ARGS__)
```

Comment:
nit, but other domain-specific logging are always "log_something", like log_device_error, log_link_error, log_unit_error, so it would be better to call these log_varlink_idl to keep things consistent

Reply:
that's not really the case, for example bus_log_xyz(), table_log_xyz(), pam_log_xyz(), json_log(), varlink_log(), …

if there's a theme here I'd claim it's precisely the other way round, subsys_log_xyz() seems to be more common that log_subsys_xyz()...